### PR TITLE
Editor can preview unpublished article

### PR DESCRIPTION
**Pull request by ViaMarcus**
[PT Card](https://www.pivotaltracker.com/story/show/173037233)
- Adds preview component
- Adds links to article list to work as selectors for what to preview
- Fix the list fetch to work against a live database (it didn't due to missing headers)
@jysmys, coauthor

### CODE REVIEW

**Comment by ViaMarcus** [comment]
Screen Shot 2020-05-30 at 19 46 08

File: src/components/Preview.jsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import { Container, Grid, Image } from "semantic-ui-react";
+import "../css/Preview.css";
+
+const Preview = ({ selectedArticle }) => {
+  return (
+    <Container id="preview" align="center">
+      <Grid>
+        <Grid.Row centered>
+          <Container id={"image"}>
```
Comment (designerofthing) [inline src/components/Preview.jsx]: why is the id in { } ?

File: src/css/Preview.css [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  height: calc(100vh - 180px);
+  overflow-x: hidden;
+  overflow-y: auto;
+}
+
+#image {
+  margin: 0;
+  padding: 0;
+  max-width: unset;
+  max-height: unset;
```
Comment (designerofthing) [inline src/css/Preview.css:18]: what does this do?

File: src/components/Preview.jsx [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+            <Image src={ selectedArticle.image }/>
+            <h2 id={"preview-title"}>{selectedArticle.title}</h2>
+          </Container>
+        </Grid.Row>
+        <Grid.Row centered id="created-text">
+          Created at: {selectedArticle.created_at}
+        </Grid.Row>
+        <Grid.Row centered id="preview-body">
+          <p id={"body"} className="article-body">
```
Comment (jysmys) [inline src/components/Preview.jsx]: just "body"

File: src/components/Preview.jsx (hunk 1 above)
Comment (erikbjoern) [reply src/components/Preview.jsx]: good question..?!

File: src/css/Preview.css (hunk 2 above)
Comment (kermit-klein) [reply src/css/Preview.css:18]: it resets any value max-width has if it is defined somewhere else

File: src/components/Review.jsx [hunk 4]
```diff
@@ -1,21 +1,25 @@
...
+          ...headers,
+          "Content-type": "application/json",
+          Accept: "application/json",
+        };
+        const response = await axios.get("/admin/articles", {
+          headers: headers,
+        });
         setUnpublishedArticleList(response.data.articles);
       } catch (error) {
         console.log(error);
```
Comment (erikbjoern) [inline src/components/Review.jsx:26]: We should probably display error messages if there would be any.

**Review by kermit-klein — APPROVED** [review]

File: src/components/Review.jsx [hunk 5]
```diff
@@ -18,27 +21,44 @@ const Review = () => {
         );
         setUnpublishedArticleList(response.data.articles);
       } catch (error) {
+        debugger;
```
Comment (PaulineBA) [inline src/components/Review.jsx]: Remove debugger ;-)

File: src/components/Preview.jsx (hunk 1 above)
Comment (ViaMarcus) [reply src/components/Preview.jsx]: Copied grid structure from singlearticle in client, it had string interpolation there, which I remove the dynamic part of. Auto formatter changed into normal string, forgot to remove {}.
Addressing this

**Review by erikbjoern — APPROVED** [review]

File: src/css/Preview.css (hunk 2 above)
Comment (ViaMarcus) [reply src/css/Preview.css:18]: The grid component was acting up a lot and it made it very narrow without this, as it for some reason set a max-width of like 400px, when i needed it to be fully responsive.

File: src/components/Review.jsx (hunk 4 above)
Comment (ViaMarcus) [reply src/components/Review.jsx:26]: Now there is a general message saying you should pick something and if you get an error this message changes to be the received error instead.

File: src/components/Review.jsx (hunk 5 above)
Comment (ViaMarcus) [reply src/components/Review.jsx]: Done

**Review by oliverochman — APPROVED** [review]

**Pull request merged**